rpcs3/Emu/Cell/SPURecompiler.cpp

```diff
@@ -5840,7 +5840,7 @@ class spu_llvm_recompiler : public spu_recompiler_base, public cpu_translator
 
 			if (auto ci = llvm::dyn_cast<llvm::ConstantInt>(trunc<u8>(val).eval(m_ir)))
 			{
-				if (g_cfg.core.spu_accurate_dma)
+				if (g_cfg.core.spu_accurate_dma || g_cfg.core.spu_debug)
```

Comment:
I really hope this change can be avoided. The ppu/spu_debug option is heavy as it is, but it's not supposed to change the execution flow so hard.

Reply:
Hmm okay.